magento/pwa-studio
# [chore] Update workbox to 5.1.2

## Description
Workbox logging is better in v5.

Review breaking changes [here](https://github.com/GoogleChrome/workbox/releases/tag/v5.0.0).

## Related Issue





Closes #issue.

## Acceptance 



### Verification Stakeholders

### Specification



### Verification Steps


1. Go to deployed page with console closed.
2. Open console. Should see no workbox error.
3. Refresh the page. Should see no workbox error.

## Screenshots / Screen Captures (if appropriate)

## Checklist

* I have updated the documentation accordingly, if necessary.
